Poker.BE/Poker.BE.Service/IServices/IAuthenticationService.cs

```diff
@@ -1,15 +1,20 @@
-﻿using System;
+﻿using Poker.BE.Service.Modules.Requests;
+using Poker.BE.Service.Modules.Results;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-
+//TODO @idan implement ipoker service
```

Comment:
you asked me to add this... i didnt have ipoker on my branch. u just need to make sure this class implements ipoker